__tests__/app.test.js

```diff
@@ -101,3 +101,33 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("returns 200 status code and an array of articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(response.status).toBe(200);
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles.length).toBe(13);
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("comment_count");
+        });
+      });
+  });
+  test("returns a 404 status code when the path is misspelled", () => {
+    return request(app)
+      .get("/api/articlessss")
```

Comment:
If you already have a test for `invalid path` then you don't need this. It's not about misspelling an endpoint but rather merely trying to access an endpoint that doesn't exist